src/ServicePulse.Host/app/js/directives/ui.particular.configurationTabs.js

```diff
@@ -0,0 +1,70 @@
+﻿
+; (function (window, angular, $, undefined) {
+    'use strict';
+
+
+    function controller($scope, $interval, $location, redirectService, notifyService, sharedDataService) {
+        var notifier = notifyService();
+
+        $scope.isActive = function (viewLocation) {
+            var active = (viewLocation === $location.path());
+            return active;
+        };
+        
+        var stats = sharedDataService.getstats();
+
+        redirectService.getTotalRedirects().then(function(response) {
+            $scope.counters.redirects = response || 0;
+        });
+
+        $scope.counters = {
+            endpoints: stats.number_of_endpoints,
+            redirects: 0
+        }
+
+        var redirectPromise = $interval(function () {
```

Comment:
This isn't a promise, it's an interval timer. Perhaps rename to `redirectUpdatedTimer`? Or something more meaningful?